packages/@react-aria/grid/src/GridKeyboardDelegate.ts

```diff
@@ -204,6 +219,15 @@ export class GridKeyboardDelegate<T, C extends GridCollection<T>> implements Key
         return item.parentKey;
       }
 
+      if (this.cycleMode === 'within') {
+        return this.direction === 'rtl' ? this.getFirstKey(key) : this.getLastKey(key);
```

Comment:
this is the opposite of the default case? asking because I actually don't know what these cycle modes are. can you add that to the PR description so I know what I'm testing?